client/eventsSDL/InputSourceKeyboard.cpp

```diff
@@ -120,15 +120,25 @@ void InputSourceKeyboard::handleEventKeyUp(const SDL_KeyboardEvent & key)
 	GH.events().dispatchShortcutReleased(shortcutsVector);
 }
 
-bool InputSourceKeyboard::isKeyboardCtrlDown() const
+bool InputSourceKeyboard::isKeyboardCmdDown() const
 {
-#ifdef VCMI_MAC
+#ifdef VCMI_APPLE
 	return SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_LGUI] || SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_RGUI];
 #else
 	return SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_LCTRL] || SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_RCTRL];
 #endif
 }
 
+bool InputSourceKeyboard::isKeyboardCtrlDown() const
+{
+#ifdef VCMI_APPLE
+	return SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_LCTRL] || SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_RCTRL] ||
+		   SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_LGUI] || SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_RGUI];
+#else
+	return SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_LCTRL] || SDL_GetKeyboardState(nullptr)[SDL_SCANCODE_RCTRL];
```

Comment:
You could extract it to a variable to reduce copy-paste

Reply:
I guess I could, but not sure if this improves readability in this case, with all these tiny methods and ifdef's

Reply:
At least you'd immediately see that both code paths handle normal Ctrl and apple just does something extra